api/src/main/resources/workbench-api.yaml

```diff
@@ -7133,6 +7133,31 @@ definitions:
         type: string
         description: 'A JSON string containing an array of JSON-encoded egress events.
           See the SumologicEgressEvent definition for the expected schema of each event.'
+  BucketAuditEntry:
+    type: object
+    required:
+      - pet_account
+      - google_project_id
+    properties:
+      pet_account:
+        description: This is the pet account associated with the user
+        type: string
+      file_lengths:
+        description: The total file lengths reported in the audit logs for the user
+        type: integer
+        format: int64
+      google_project_id:
+        description: The google project id that caused the egress
+        type: string
+      bucket_name:
+        description: The name of the bucket reported in the audit logs that contains the files
+        type: string
+      min_time:
```

Comment:
(no action required, but something to consider). I have seen we use timestamp this way: 
https://github.com/all-of-us/workbench/blob/main/api/src/main/resources/workbench-api.yaml#L1452

Reply:
thank you Yonghao, yea I tried this but the timestamp that we get back from bigquery is easily handled as string or localdatetime, when I tried to get it back as a number, it was not obvious what the format is so I was not able to convert it back to datetime.